tests/integration_tests/allennlp_tests/test_allennlp.py

```diff
@@ -107,3 +110,25 @@ def test_allennlp_executor_with_include_package_arr() -> None:
         )
         result = executor.run()
         assert isinstance(result, float)
+
+
+def test_dump_best_config() -> None:
+    with tempfile.TemporaryDirectory() as tmp_dir:
+
+        def objective(trial: optuna.Trial) -> float:
+            trial.suggest_uniform("DROPOUT", dropout, dropout)
+            executor = optuna.integration.AllenNLPExecutor(trial, input_config_file, tmp_dir)
+            return executor.run()
+
+        dropout = 0.5
+        input_config_file = "tests/integration_tests/allennlp_tests/example.jsonnet"
```

Comment:
How about deriving the directory instead of hard-coding it from the root?
```suggestion
        input_config_file = os.path.join(
            os.path.dirname(os.path.realpath(__file__)), "example.jsonnet"
        )
```